usr.bin/unzip/unzip.1

```diff
@@ -34,9 +34,10 @@
 .Sh SYNOPSIS
 .Nm
 .Op Fl aCcfjLlnopqtuvy
+.Op { Fl O | Fl I No } Ar encoding
 .Op Fl d Ar dir
 .Op Fl x Ar pattern
-.Op Fl P Ar password
+.Op Fl P Ar password 
```

Comment:
this looks like it adds trailing white space and should be removed.

Reply:
Oops!